src/Views/FirmwareView.vala

```diff
@@ -120,12 +120,12 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
                 add_device (devices[i]);
             }
 
-            placeholder_alert_view.title = _("Firmware Updates Are Not Available");
-            placeholder_alert_view.description = _("Firmware updates are not supported on this or any connected devices.");
+            placeholder_alert_view.title = _("No new firmware available");
+            placeholder_alert_view.description = _("None of your devices support installing new firmware this way.");
             update_list.show_all ();
         } catch (Error e) {
-            placeholder_alert_view.title = _("The Firmware Service Is Not Available");
-            placeholder_alert_view.description = _("Please make sure “fwupd” is installed and enabled.");
+            placeholder_alert_view.title = _("Firmware service unvailable");
```

Comment:
Spelling: "unvailable! should be "unavailable"

Reply:
Good catch!

Reply:
```suggestion
            placeholder_alert_view.title = _("Firmware service unavailable");
```